acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/Node.java

```
@@ -60,6 +61,12 @@ protected Node(final Network network, final String dockerImageName, final Logger
             .withLogConsumer(frame -> log.debug(frame.getUtf8String().trim()));
   }
 
+  protected Node(final Network network, final ImageFromDockerfile image) {
+    this.nodeAlias =
+        getClass().getSimpleName().toLowerCase(Locale.US) + NODE_UNIQUIFIER.incrementAndGet();
+    this.container = new NodeContainer(image).withNetwork(network).withNetworkAliases(nodeAlias);
+  }
```

I suspect we should add a log consumer like in the original constructor here - otherwise we will silently ignore any log output from the container which makes debugging failures difficult.